### PR TITLE
entrez & uniprot links added

### PR DESCRIPTION
**Pull request by Nikhil-lab**
- entrez & uniprot links added to the info component.
- asctb-api/src/functions/lookup.functions.ts updated by adding two new functions for building links
- lasctb-api/src/models/lookup.model.ts is updated
- Backend response is updated accordingly

### CODE REVIEW

**Review by bherr2 — CHANGES_REQUESTED** [review]
Please fix bugs / make changes requested from Sonar: https://sonarcloud.io/project/issues?id=hubmapconsortium_ccf-asct-reporter&pullRequest=231&resolved=false&types=BUG and lint errors brought out from testing: https://github.com/hubmapconsortium/ccf-asct-reporter/pull/231/checks?check_run_id=3680016592#step:6:12. Run `npm run lint` and `npm run test` locally to verify.

File: projects/v2/src/app/components/info/info.component.html [hunk 1]
```diff
@@ -69,15 +69,15 @@
       <div *ngIf="info.entrezLink">
       <div class="pt-3"></div>
 
-      <label ><b>Entrez Link</b></label>
+      <label ><strong>Entrez Link</strong></label>
```
Comment (bherr2) [inline projects/v2/src/app/components/info/info.component.html:67]: Generalize this to an *ngFor over Object.items(extraLinks). The idea was that extra links could be provided (and doesn't need to be hard coded here).

File: projects/v2/src/app/services/sheet.service.ts [hunk 2]
```diff
@@ -81,8 +81,8 @@ export class SheetService {
         ontologyCode,
         desc: res.description,
         iri: res.link,
-        entrezLink: res.extraLinks["Entrez Link"],
-        uniprotLink: res.extraLinks["Uniprot Link"],
+        entrezLink: res.extraLinks['Entrez Link'],
```
Comment (bherr2) [inline projects/v2/src/app/services/sheet.service.ts:84]: Should just need to track 'extraLinks', not hard code uniprot + entrez

**Review by bherr2 — CHANGES_REQUESTED** [review]
Please generalize so that we're not hard coding entrez + uniprot on the client side. There could easily be more links added at a later time.

File: projects/v2/src/app/services/sheet.service.ts (hunk 2 above)
Comment (Nikhil-lab) [reply projects/v2/src/app/services/sheet.service.ts:84]: ok Bruce. Got it. Let me do the necessary changes.

**Comment by bherr2** [comment]
@Nikhil-lab lint errors: https://github.com/hubmapconsortium/ccf-asct-reporter/pull/231/checks?check_run_id=3724092048#step:8:27

File: projects/v2/src/app/components/info/info.component.html [hunk 3]
```diff
@@ -39,31 +39,46 @@
...
       <div class="pt-3"></div>
 
-      <label><b>{{ info.ontologyCode === 'HGNC' ? 'Link' : 'IRI'}}</b></label>
+      <label><strong>{{ info.ontologyCode === 'HGNC' ? 'Link' : 'IRI'}}</strong></label>
       <div>
         <a [href]="info.iri" target="_blank" class="text-info">{{info.iri ? info.iri : 'No IRI found.'}}</a>
       </div>
 
-      <div class="pt-3"></div>
+    <div *ngIf="info.ontologyCode === 'HGNC'">
```
Comment (bherr2) [inline projects/v2/src/app/components/info/info.component.html:63]: This is not specific to HGNC

Comment (bherr2) [reply projects/v2/src/app/components/info/info.component.html:63]: Other lookups can add extraLinks

File: asctb-api/src/routes/ontology-lookup.ts [hunk 4]
```diff
@@ -17,18 +22,19 @@ export function setupOntologyLookupRoutes(app: Express): void {
...
-
         res.send({
+          extraLinks: {
+            'Uniprot Link': buildUniprotLink(firstResult.uniprot_ids[0]),
+            'Entrez Link': buildEntrezLink(firstResult.entrez_id),
+          },
           label: firstResult.symbol,
           link: buildHGNCLink(firstResult.hgnc_id),
-          description: firstResult.name ? firstResult.name : ''
+          description: firstResult.name ? firstResult.name : '',
```
Comment (bherr2) [inline asctb-api/src/routes/ontology-lookup.ts:36]: no trailing commas

**Review by bherr2 — CHANGES_REQUESTED** [review]

File: asctb-api/src/routes/ontology-lookup.ts (hunk 4 above)
Comment (Nikhil-lab) [reply asctb-api/src/routes/ontology-lookup.ts:36]: trailing commas were added by prettier extension in VS code. I will remove it .

File: projects/v2/src/app/components/info/info.component.html (hunk 3 above)
Comment (Nikhil-lab) [reply projects/v2/src/app/components/info/info.component.html:63]: ok got it.

**Comment by bherr2** [comment]
Still some remaining trailing commas (look at the files tab). There are also merge conflicts, so please pull from `develop`, fix, and push.

**Review by bherr2 — APPROVED** [review]

**Pull request merged**